packages/meditrak-server/src/routes/exportSurveyResponses.js

```diff
@@ -68,7 +68,7 @@ export async function exportSurveyResponses(req, res) {
     latest = false,
     startDate,
     endDate,
-    timeZone,
+    timeZone = 'Australia/Melbourne',
```

Comment:
I'm not sure if we should provide a default here or let it fail. I'd also like @edmofro 's opinion on that, but for now let's use `UTC` as a default since it matches our server's timezone

Reply:
In other places we use Auckland as the default, because timezone wise it aligns more closely with Pacific Island Countries we work in than Melbourne. I think UTC is fine here though, because it's more predictable than anything specific, and won't actually be triggered under our current use cases.

Great fix btw @chris-bes! It seems it's about time to put together some more formal regression matrices.